README.md

```diff
@@ -1 +1,39 @@
-# HelloWorld
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```

Comment:
- [ ] Update the project title in the readme. It's important you customize the readme to your project.